azurerm/internal/services/compute/windows_virtual_machine_scale_set_resource.go

```diff
@@ -146,6 +146,13 @@ func resourceArmWindowsVirtualMachineScaleSet() *schema.Resource {
 
 			"extension": VirtualMachineScaleSetExtensionsSchema(),
 
+			"extensions_time_budget": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				Default:      "PT1H30M",
```

Comment:
Is this always the default when none is specified? or should we make this computed?